src/Pages/Login/Login.js

```diff
@@ -53,14 +50,24 @@ class Login extends React.Component {
 
       const result = await res.json();
 
-      console.log(result);
+      if (result.token) {
+        localStorage.setItem('access_token', result.token);
```

Comment:
```suggestion
        setToken(result.token);
```
이런 식으로 사용할 수 있도록 src/utils에 함수 작성하면 어떨까요? getToken()도 마찬가지로요!

Reply:
확인하고 수정했습니다!